api/openapi.gen.json

```diff
@@ -331,7 +331,14 @@
               "id": "lt-9843797432897342",
               "name": "XXL large backend API"
             }
-          ]
+          ],
+          "links": {
+            "next": "",
+            "previous": ""
+          },
+          "metadata": {
+            "total": 0
```

Comment:
Not sure why it created the total here if it is empty and I am using the omitempty tag. 
Wanted to change the metadata struct to pointer but it is the same discussion as on Slack